lms/static/sass/discussion/_discussion.scss

```diff
@@ -71,7 +59,7 @@ body.discussion {
 
   .thread-title {
     display: block;
-    margin-bottom: $baseline;
+    margin-bottom: ($baseline);
```

Comment:
That was a mistake on my part. I had to add "+ px" to every line like this because the Pattern Library didn't include it, but then that in turn caused problems for the legacy non-Pattern Library pages. I went back and removed the suffix but then left the unnecessary parens. I've removed them now.